hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -57,6 +57,7 @@
 public class RDBStore implements DBStore {
   private static final Logger LOG =
       LoggerFactory.getLogger(RDBStore.class);
+  public static final int MAX_DB_UPDATES_SIZE_THRESHOLD = 1024 * 1024;
```

Comment:
Shouldn't this be 1024 * 1024 * 1024?

Reply:
> 

@jojochuang - Thanks for the review. Yes you are right, I have changed that to 1024 * 1024 * 1024.